.github/actions/prepush_release_hook/action.yml

```diff
@@ -0,0 +1,18 @@
+inputs:
+  semver:
+    required: true
+    type: string
+  tag:
+    required: true
+    type: string
+
+runs:
+  using: composite
+  steps:
+  - name: Zip submitter plugin
+    shell: bash
+    run: |
+      mkdir dist_extras
+      cd src/deadline/blender_submitter/addons
+      zip -r  ../../../../dist_extras/blender_submitter_plugin_${{inputs.semver}}.zip .
+      cd ../../../../
```

Comment:
nit: can use pushd/popd here, though I don't think you need to do the last cd when it's encapsulated in the bash script. Doesn't actually affect anything here though.